Components/Avatars/avatar.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./../../css/main.css">
+    <link rel="stylesheet" href="./../../css/components.css">
+    <link rel="stylesheet" href="./avatar.css">
+
+    <title>minimali ui: Avatars</title>
+</head>
+
+<body class="bg-white">
+
+    <nav class="navbar">
+        <h2 class="text-2xl ff-title fw-semibold">minimali.</h2>
+        <ul class="navbar-right text-normal">
+            <li><a class="links" href="">Home</a></li>
+            <li><a class="links" href="">Docs</a></li>
+            <li><a class="links" href="">Github</a></li>
+        </ul>
+    </nav>
+
+    <main class="main-container">
+        <section class="side-bar">
+
+            <!-- Introduction List Items -->
+            <div class="">
+                <h3 class="fw-bold text-large">Getting Started</h3>
+                <ul class="intro-list text-sm">
+                    <li>
+                        <a class="links" href="#">Introduction</a>
+                    </li>
+                    <li>
+                        <a class="links" href="#">Colors</a>
+                    </li>
+                    <li>
+                        <a class="links" href="#">Typography</a>
+                    </li>
+                </ul>
+            </div>
+
+            <!-- Components List Items -->
+            <div class="">
```

Comment:
If class is there, you can remove class = ""

Reply:
Sure will make the changes